README.md

```diff
@@ -12,25 +12,23 @@ Lingo is a Rust project and is built with cargo. To install it simply run
 ## The command line interface
 
 ```
-lingua-franca package manager and build tool 0.1.2
-tassilo.tanneberger@tu-dresden.de
-Build system of lingua-franca projects
-
-USAGE:
-    lingo [OPTIONS] <SUBCOMMAND>
-
-OPTIONS:
-    -b, --backend <BACKEND>    Force lingo to use the specified backend [default: cli]
-    -h, --help                 Print help information
-    -V, --version              Print version information
-
-SUBCOMMANDS:
-    build     Compile the current package
-    clean     Remove build artifacts from the package
-    help      Print this message or the help of the given subcommand(s)
-    init      Initialize a new package
-    run       Build and run a main program in the package
-    update    Update all the dependencies in the package
+Build system for the Lingua Franca coordination language
+
+Usage: lingo [OPTIONS] <COMMAND>
+
+Commands:
+  init    initializing a lingua-franca project
+  build   compiling one or multiple binaries in a lingua-franca package
+  update  Updates the dependencies and potentially build tools
```

Comment:
I put this in imperative form, but it's still not entirely clear to me what the "potentially build tools" means.
```suggestion
  update  Update the dependencies and potentially build tools
```